src/cli.rs

```diff
@@ -8,6 +8,10 @@ pub struct Args {
     /// Enabled by default if the app is running under `cargo leptos ...`
     #[arg(short = 'n', long = "no-open", default_value_t = in_cargo_leptos())]
     pub no_open: bool,
+
+    /// Provides a way to control the address leptos is served from
+    #[arg(short = 's', long = "site-addr")]
+    pub leptos_site_addr: Option<String>,
```

Comment:
Isn't there a way to make this `Option<SocketAddr>`?

Reply:
yes, that works, I have updated the PR